assets/stylesheets/foreground.css

```diff
@@ -522,7 +523,7 @@ button, .button {
 }
 
 .top-bar .button.search {
-	top: 0px;
+	top: -1px;
```

Comment:
Why did you add a -1px, that breaks alignment.

![searchbuttonalignmenttop](https://cloud.githubusercontent.com/assets/3527469/12053042/1313fba0-aee3-11e5-80b1-dc47d7a620a1.PNG)